### ISSUE TITLE
Problem: S3 is not supported for AIP Storage 

### ISSUE THREAD

**Issue by jhsimpson** [artefactual/archivematica-storage-service#339]
The Storage Service has Spaces defined for DuraCloud and for Swift, but not for Amazon S3. 

It should be possible to store AIP's in S3.  Ideally the way this is implemented would be as consistent as possible across both S3 and Swift.

It should be possible to store both compressed and uncompressed aips.

Ideally the new move_direct method of transfering packages between locations would be supported.

Other kinds of Storage Locations (such as Transfer Source and DIP Storage) should eventually also be supported by AIP Storage is the highest priority.

**Comment by nickwilkinson** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Jisc RDSS](https://trello.com/c/LGTIS5UR/9-jisc-rdss)

**Comment by joel-simpson** [comment]
This is connected to #342 (already shown in Github but not appearing in Waffle for some reason?)
I have tested creating many AIPS now of several types (standard vs. Bag, vs. Zipped Bag etc.; compressed and uncompressed). I have confirmed the AIPs can be searched for in Archival Storage Tab, or found by browsing in the Storage Service. confirmed the AIPs can be downloaded and opened. Pointer files are correctly created for compressed AIPs (but not uncompressed). 
Testing in qa/1.x as of July 25, on Ubuntu using Docker containers. 
All looks good so moving this to 'verified'.
